subprojects/docs/src/docs/release/notes.md

```diff
@@ -50,6 +50,26 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+### Faster Java compilation
+
+When compiling Java code, Gradle uses worker processes to run the Java compiler as a compiler daemon. 
+Compiler daemons are reused within a single build invocation. 
+This allows the Java compiler to "warm-up" and compile faster after a few uses. 
+
+In previous Gradle releases, the compiler daemons were always shut down at the end of the build.
+This caused every build to incur the cost of starting the compiler daemon and warming up the compiler.
+These start-up costs contributed to a large portion of overall build time when incrementally compiling a few source files.
+
+Starting in this release, Gradle attempts to keep Java compiler daemons around after the end of the build, so that subsequent builds are faster.
```

Comment:
Can we use a different word for `around`?
```suggestion
Starting in this release, Gradle attempts to keep Java compiler daemons alive after the end of the build, so that subsequent builds are faster.
```